SecretManager/src/V1/Client/SecretManagerServiceClient.php

```diff
@@ -151,6 +168,82 @@ public static function projectName(string $project): string
         ]);
     }
 
+    /**
+     * Formats a string containing the fully-qualified path to represent a
+     * project_location_secret resource.
+     *
+     * @param string $project
+     * @param string $location
+     * @param string $secret
+     *
+     * @return string The formatted project_location_secret resource.
+     */
+    public static function projectLocationSecretName(string $project, string $location, string $secret): string
+    {
+        return self::getPathTemplate('projectLocationSecret')->render([
+            'project' => $project,
+            'location' => $location,
+            'secret' => $secret,
+        ]);
+    }
+
+    /**
+     * Formats a string containing the fully-qualified path to represent a
+     * project_location_secret_secret_version resource.
+     *
+     * @param string $project
+     * @param string $location
+     * @param string $secret
+     * @param string $secretVersion
+     *
+     * @return string The formatted project_location_secret_secret_version resource.
+     */
+    public static function projectLocationSecretSecretVersionName(string $project, string $location, string $secret, string $secretVersion): string
```

Comment:
I think it would be better to name it as `projectLocationSecretVersionName` instead of `projectLocationSecretSecretVersionName`!

Reply:
cc: @bshaffer

Reply:
This is auto-generated, and it is created from the URL template by concatenating all the variable names in the URL. In this case, the URL template is:

```
 - projectLocationSecretSecretVersion: projects/{project}/locations/{location}/secrets/{secret}/versions/{secret_version}
```

As you can see, because the last variable is called `secret_version`, and the one before it is called `secret`, the method name ends up having `SecretSecret`. This is not ideal, but the fix for this would require changing the variable name in the API (it's too late for this), or writing a custom fix in the generator (this is hacky and error prone).

Reply:
Hmm.. Will merge this for now then, thank you Brent for the confirmation.